### ISSUE TITLE
Extension keeps changing .wakatime-project file

### ISSUE THREAD

**Issue by petoem** [wakatime/vscode-wakatime#61]
Wakatime vscode extension keeps writing new project name into `.wakatime-project`. 

![image](https://user-images.githubusercontent.com/8493004/43260010-6e46c118-90d8-11e8-8a01-89543dac4f15.png)

`.wakatime.log` shows this error:
```json
{
  "now": "2018/07/26 12:57:34 +0200",
  "version": "10.2.1",
  "plugin": "vscode/1.25.1 vscode-wakatime/1.2.3",
  "time": 1532602653.8137739,
  "caller": "/home/petoem/.vscode/extensions/wakatime.vscode-wakatime-1.2.3/out/wakatime-master/wakatime/logger.py",
  "lineno": 62,
  "file": "/home/petoem/Documents/workspace/awesomeproject/.wakatime-project",
  "level": "DEBUG",
  "message": "Traceback (most recent call last):\n  File \"/home/petoem/.vscode/extensions/wakatime.vscode-wakatime-1.2.3/out/wakatime-master/wakatime/stats.py\", line 123, in guess_lexer_using_filename\n    lexer = custom_pygments_guess_lexer_for_filename(file_name, text)\n  File \"/home/petoem/.vscode/extensions/wakatime.vscode-wakatime-1.2.3/out/wakatime-master/wakatime/stats.py\", line 316, in custom_pygments_guess_lexer_for_filename\n    raise ClassNotFound('no lexer for filename %r found' % fn)\npygments.util.ClassNotFound: no lexer for filename '.wakatime-project' found\n"
}
```

My `.wakatime.cfg`:
```ini
[settings]
debug = true
api_key = 
hide_file_names = true
hide_project_names = true
; include_only_with_project_file = true
offline = true
timeout = 15
hostname = notebook
```

**Comment by alanhamlett** [comment]
The WakaTime plugin only creates that DEBUG log when VS Code triggers these editor events:
https://github.com/wakatime/vscode-wakatime/blob/0efd068fd24c2c5638a96e766c3b91711f4e8afb/src/extension.ts#L235

Maybe clicking on the `.wakatime-project` file inside your project folder is triggering those events, or another plugin is causing unopened files to be modified or opened within your project?

**Comment by alanhamlett** [comment]
Fixed with wakatime/wakatime@8afc6133cf2645d95c0acc84fcddcb5493e508c2 and released just now in v10.7.1. Restarting your VS Code should fix it.

**Comment by hilleer** [comment]
I have just set `hide_project_names` to true today. As a result Wakatime started generating those files as suggested by @edmondburnett...

Its a bit silly to add those to all my project's `.gitignore`, while I believe it is unexpected behaviour?
